docs/guides/payments.rst

```diff
@@ -232,7 +226,7 @@ Parameters
 +-------------------------+-----------------+-----------------------------------------------------------------------------+
 | ``payment_information`` | ``PaymentData`` | Payment information, containing the token, amount, currency and billing.    |
 +-------------------------+-----------------+-----------------------------------------------------------------------------+
-| ``connection_params``   | ``dict``        | List of parameters used for connecting to the payment's gateway.            |
+| ``config``              | ``ConfigData``  | Configuration of the payment gateway.                                               |
```

Comment:
nit: isn't last pipe (`|`) too far?

Reply:
Looks like it

<img width="1054" alt="Screenshot 2019-05-14 at 12 21 02" src="https://user-images.githubusercontent.com/6186720/57690803-c8c84d00-7642-11e9-84b9-29f52e326158.png">